packages/ui/src/ui/pages/navigation/Navigation/NavigationTopRowContent.tsx

```diff
@@ -155,15 +156,32 @@ function NavigationPathEditor({hideEditor}: {hideEditor: () => void}) {
 
 function NavigationBreadcrumbs({onEdit}: {onEdit: () => void}) {
     const bcItems = useSelector(getNavigationBreadcrumbs);
+    const mode = useSelector(getMode);
+    const history = useHistory();
     const items = React.useMemo(() => {
         return _.map(bcItems, ({text, state}) => {
+            const url = makeRoutedURL(window.location.pathname, {
+                path: state.path,
+                navmode: mode === Tab.ACL ? mode : Tab.CONTENT,
+                filter: '',
+            });
+
             return {
                 text,
-                state: state,
-                action: () => {},
+                url,
+                state,
+                action: (event: React.MouseEvent<HTMLElement, MouseEvent> | KeyboardEvent) => {
+                    event.preventDefault();
+                    event.stopPropagation();
+                    if (!event.ctrlKey && !event.metaKey) {
```

Comment:
Another way to open new tab is using of middle button click.

Reply:
Sure, I added `&& event.button !== 1` condition.